### PR TITLE
sshconfig: get operator home directory via getent passwd

### LINKED ISSUE #352: sshconfig: get operator home directory via getent passwd
```
# NOTE: http://stackoverflow.com/questions/33343215/how-to-get-remote-users-home-directory-in-ansible

- name: Get home directory of operator user
  become: true
  ansible.builtin.shell: |
    set -o pipefail
    getent passwd {{ operator_user }} | cut -d: -f6
  args:
    executable: /bin/bash
  changed_when: false
  register: operator_home
```

### PR DESCRIPTION
**Pull request by ra-rau**
Closes #352

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: roles/sshconfig/README.rst [hunk 1]
```diff
@@ -14,7 +14,6 @@ The user who will own the configuration directory.
 The group which will own th econfiguration directory.
 
 .. zuul:rolevar:: sshconfig_destination
```
Comment (berendt) [inline roles/sshconfig/README.rst:16]: Remote this as well

File: roles/sshconfig/README.rst [hunk 2]
```diff
@@ -14,7 +14,6 @@ The user who will own the configuration directory.
...
-   :default: /home/{{ operator_user }}/.ssh
 
 Directory where the configuration file will be stored in.
```
Comment (berendt) [inline roles/sshconfig/README.rst:18]: Remove this as well

File: roles/sshconfig/README.rst (hunk 1 above)
Comment (ra-rau) [reply roles/sshconfig/README.rst]: Done

File: roles/sshconfig/README.rst (hunk 2 above)
Comment (ra-rau) [reply roles/sshconfig/README.rst]: Done

**Pull request merged**